### DISCUSSION TITLE
Fixed export price not showing correctly

### DISCUSSION THREAD

**Discussion by MrRusti** [netzero-labs/netzero#62] [Q&A]
I hope this is the right place to post, but I’ve searched here and on the Tesla forum (TMC) and can’t find any info to help with my query.
I’ve been using NetZero on Android for a few months to automate my PW2and love the app.
I’m on Octopus Agile for import and Fixed for export (15p per kwhr) and these are set fine in the NetZero app.
NetZero says export tariff is "outgoing 12M Fixed May 2019", lowest = highest = 15p.
But the Grid page of Tesla app shows variable export prices.  (screenshot attached)
I’ve tried changing the export tariff, & turning the auto update off & on again, but the Tesla Grid page continues to show variable export prices.
Can you help me get the correct (fixed) export price updated in the Tesla app ?
Many thanks.
![Tesla app 1](https://github.com/user-attachments/assets/5427908e-ba0d-4a3b-87d5-10675721656a)

**Comment by zigam** [comment]
Tesla has a restriction in Time-Based Control mode where the sell price cannot exceed the buy for any period. Netzero will adjust the sell price downward if necessary, which results in improved Powerwall behavior. The next app version will add a note about this to the Utility Rate Plan screen in Netzero, to avoid confusion.

More about dynamic tariffs: https://www.netzero.energy/docs/tariffs

**Comment by MrRusti** [comment]
Thanks for the quick reply, I get it now.
Looking forward to the next app version.
